PublicationsRouter/v2/sword-out/pubrouter-xml/pubrouter.xsd

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+    targetNamespace="http://pubrouter.jisc.ac.uk/rioxxplus"
+    xmlns:rioxxterms="http://www.rioxx.net/schema/v2.0/rioxxterms/"
+    xmlns:ali="http://ali.niso.org/2014/ali/1.0"
+    xmlns:dcterms="http://purl.org/dc/terms/"
+    xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning" vc:minVersion="1.1"
+    xmlns="http://pubrouter.jisc.ac.uk/rioxxplus">
+    <xs:import namespace="http://www.rioxx.net/schema/v2.0/rioxxterms/"
+        schemaLocation="http://rioxx.net/schema/v2.0/rioxx/rioxxterms.xsd"/>
+    <xs:import namespace="http://ali.niso.org/2014/ali/1.0"
+        schemaLocation="http://rioxx.net/schema/v2.0/rioxx/ali.xsd"/>
+    <xs:import namespace="http://purl.org/dc/terms/"
+        schemaLocation="http://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd"/>
+
```

Comment:
Please can you add <annotation> elements to describe the Schema.

Need an Overview annotation that explains the purpose of the schema - ie. to provide maximum PubRouter meta-data information.

I think each element should also have a brief annotation to describe it. By default this can be the text in the text in the Notes column.